tests/integration/targets/azure_rm_virtualmachine_scaleset/tasks/main.yml

```diff
@@ -240,86 +258,6 @@
   assert:
     that: not results.changed
 
-- name: Install VMSS Extension
```

Comment:
I think this should be not removed. it's in pr upstream

Reply:
added back